app/feedpeople/tests.py

```diff
@@ -0,0 +1,14 @@
+from django.test import TestCase
+
+from .models import FeedPerson
+
+# Create your tests here.
+class FeedPersonTestCase(TestCase):
+
+  def setUp(self):
+    feed_person = FeedPerson.objects.create(eppn="00000000", firstname="John", lastname="Harvard", location="Pierce Hall 101", name="John Harvard")
```

Comment:
Minor point, but can we use something more eppn-like for the test eppn value, i.e 16-character hexadecimal string like:

```
1a2b3c456def7890
```

Reply:
Ah yes I was getting HUID mixed with eppn. This was updated in 2c78b55.